sdk/openai/openai/test/utils/utils.ts

```diff
@@ -75,18 +84,24 @@ export async function withDeployments<T>(
             "ModelDeprecated",
             "429",
             "UserError",
-            400,
           ].includes(error.code) ||
           error.type === "invalid_request_error" ||
           error.name === "AbortError" ||
           errorStr.includes("Connection error") ||
           errorStr.includes("toolCalls") ||
-          error.status === 404
+          [
```

Comment:
nit: Would it make sense to put all these strings in an `AcceptableErrors` variable for this list similar to `GlobalAcceptedErrors`? Feel free to ignore this as we're moving away from this function.

Reply:
Let's attempt to address this in a future PR.